src/shim/bigint.rs

```diff
@@ -3,14 +3,24 @@
 
 use std::ops::{Deref, DerefMut};
 
-pub use fvm_shared3::bigint::bigint_ser::{BigIntDe, BigIntSer};
+pub use fvm_shared::sector::StoragePower as StoragePowerV2;
```

Comment:
`StoragePowerV2` and `StoragePowerV3` are the same type. Let's just export `StoragePower = BigInt`.

Reply:
Fixed